tests/test_code_review_fixes.py

```
@@ -0,0 +1,604 @@
+        """Test that empty rows trigger proper warnings."""
+        from xerparser.reader import Reader
+
+        # Create a temp XER file with an empty row
+        with tempfile.NamedTemporaryFile(mode='w', delete=False, suffix='.xer', encoding='utf-8') as f:
+            f.write("ERMHDR\t8.0\t2021-11-02\tProject\n")
+            f.write("\n")  # Empty row
+            f.write("%E\n")
+            temp_file = f.name
+
+        try:
+            with warnings.catch_warnings(record=True) as w:
```

Ruff (0.14.13)

Local variable `w` is assigned to but never used

Remove assignment to unused variable `w`

(F841)